Individual Scripts/windows 10 service disabler.ps1

```diff
@@ -0,0 +1,112 @@
+    "fhsvc"                                        #Disables fax histroy
+    "gupdate"                                      #Disables google update
+    "gupdatem"                                     #Disable another google update
+    "stisvc"                                       #Disables Windows Image Acquisition (WIA)
+    "AJRouter"                                     #Disables (needed for AllJoyn Router Service)
+    "MSDTC"                                        # Disables Distributed Transaction Coordinator
+    "dmwappushservice"                             #Device Management Wireless Application Protocol (WAP) Push message Routing Service
+    "WpcMonSvc"                                    #Disables Parental Controls
+    "PhoneSvc"                                     #Disables Phone Service(Manages the telephony state on the device)
+    "PrintNotify"                                  #Disables Windows printer notifications and extentions
+    "PcaSvc"                                       #Disables Program Compatibility Assistant Service
+    "WPDBusEnum"                                   #Disables Portable Device Enumerator Service
+    "LicenseManager"                               #Disable LicenseManager(Windows store may not work properly)
+    "seclogon"                                     #Disables  Secondary Logon(disables other credentials only password will work)
+    "SysMain"                                      #Disables sysmain
+    "lmhosts"                                      #Disables TCP/IP NetBIOS Helper
+    "wisvc"                                        #Disables Windows Insider program(Windows Insider will not work)
+    "FontCache"                                    #Disables Windows font cache
+    "RetailDemo"                                   #Disables RetailDemo whic is often used when showing your device
+    "ALG"                                          # Disables Application Layer Gateway Service(Provides support for 3rd party protocol plug-ins for Internet Connection Sharing)
+    #"BFE"                                         #Disables Base Filtering Engine (BFE) (is a service that manages firewall and Internet Protocol security)
+    #"BrokerInfrastructure"                         #Disables Windows infrastructure service that controls which background tasks can run on the system.
+    "SCardSvr"                                      #Disables Windows smart card
+    "lfsvc"                                         #Disable Windows geolocation service  it can be use to track you
+    "EntAppSvc"                                     #Disables enterprise application management.
+    "BthAvctpSvc"                                   #Disables AVCTP service (if you use  Bluetooth Audio Device or Wireless Headphones. then don't disable this)
+    "FrameServer"                                   #Disables Windows Camera Frame Server(this allows multiple clients to access video frames from camera devices.)
```

Comment:
My camera did not work in Zoom, which I am sure is important to most people. (Should be left enabled by default)